### PR TITLE
API-1319-batchform-fix

### PR DESCRIPTION
**Pull request by matt-morales**
handle form elements without `key`
+ don't try to change attribute on undefined item

Just check out `src/cn-batch-forms.service.js`

Also, I accidentally cut my branch from master. I will create a release for wildcat. Is it necessary for me to merge this into dev first?

### CODE REVIEW

**Review by davezuch — APPROVED** [review]
> handle form elements without

I don't really understand this, but the changes make sense.

> Also, I accidentally cut my branch from master. I will create a release for wildcat. Is it necessary for me to merge this into dev first?

I think this has to be merged into dev in order for the changes to make it to staging (because that process merges dev into staging), but the release process may have changed a lot in the time since I started working on OMS so that may not still be true.

**Comment by matt-morales** [comment]
sorry @davezuch I meant to say it handles form elements without the `key` attribute. For instance:
```
{"type": "helptext", "description": "Content Source"}
```

**Comment by matt-morales** [comment]
Ok, I rebased the branch

**Comment by davezuch** [comment]
> sorry @davezuch I meant to say it handles form elements without the `key` attribute. For instance:
> 
> ```
> {"type": "helptext", "description": "Content Source"}
> ```

That makes sense, thanks for explaining!

**Review by jocteau — APPROVED** [review]

**Comment by jojordan3** [comment]
👀 
cools

File: dist/all.js [hunk 1]
```diff
@@ -18272,55 +18274,55 @@ function cnBatchForms(cnFlexFormConfig, cnFlexFormService, cnFlexFormTypes, sfPa
...
+		Object.defineProperty(module, "id", {
+			enumerable: true,
+			get: function() {
+				return module.i;
+			}
+		});
+		module.webpackPolyfill = 1;
+	}
+	return module;
+};
```
Comment (jojordan3) [inline dist/all.js:18325]: is it me or does this issue happen way for often with js than python? (finding a diff where there is none)

**Pull request merged**